2019/pt-br/src/0xa7-security-misconfiguration.md

```diff
@@ -0,0 +1,80 @@
+API7:2019 Má Configuração de Segurança
+======================================
+
+| Agentes/Vetores | Fraquezas de Segurança | Impactos |
+| - | - | - |
+| Específico da API : Explorabilidade **3** | Prevalência **3** : Detecção **3** | Técnico **2** : Específico do negócio |
+| Atacantes eventualmente procuram falhas não corrigidas, *endpoints* comuns, ou diretórios não protegidos para ganhar acesso não-autorizado ou realizar um reconhecimento do sistema. | Configurações inadequadas de segurança podem ocorrer a qualquer nível do *stack* da API, desde o nível da rede até o nível da aplicação. Ferramentas de automação estão disponíveis para detectar e explorar erros de configuração, como serviços desnecessários e opções de suporte ao legado. | Configurações inadequadas podem não apenas expor dados sensíveis de usuários, como também podem revelar detalhes do sistema e comprometer servidores como um todo. |
+
+## A API está vulnerável?
+
+Sua API pode estar vulnerável se:
+
+* Configurações apropriadas de *hardening* faltam a qualquer parte da aplicação, ou quando há permissões mal configuradas em provedores de nuvem.
+* Os últimos *patches* de segurança não estão aplicados ou o sistemas estão desatualizados.
```

Comment:
fix
* where it's written "_(...) ou **o** sistemas estão desatualizados._"
* it should be "_(...) ou **os** sistemas estão desatualizados._"